{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-bleak-retry-connector",
	"source": "bleak-retry-connector",
	"version": "3.9.0-1",
	"section": "universe/python",
	"priority": "optional",
	"architecture": "all",
	"depends": [
		"python3-async-timeout | python3-supported-min (>= 3.11)",
		"python3-bleak",
		"python3-bluetooth-adapters",
		"python3-dbus-fast",
		"python3:any"
	],
	"installedSize": 73,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "Connector for Bleak Clients that handles transient connection failures",
	"homepage": "https://github.com/bluetooth-devices/bleak-retry-connector",
	"filename": "pool/universe/b/bleak-retry-connector/python3-bleak-retry-connector_3.9.0-1_all.deb",
	"size": 15344,
	"md5": "b113c17feb890f93951edf0ef8d5c0fb",
	"sha1": "575fad441aa1be0fd64b9fe52e6af2ed82816968",
	"sha256": "6862c61c8bc1a84195f4b484b1ced5a7f68df2d4098bd548ba40571d00f3ff2b",
	"sha512": "48cfc4160905cdef4e7c3d09296c809c3b933ae0039d7c3e7b33badb3a143174020686c446a58369984f675ed8424746c3dbe32786b8659a0b60ca18f075b663",
	"descriptionMd5": "94cf3179d80dc95dc44c0b30cf94cbad"
}